contents/docs/advanced/cdp.md

```diff
@@ -37,9 +37,9 @@ In general, we recommend starting with PostHog as your CDP using our SDKs, data
 
 If you need destinations that we don't currently support, you have several options
 
-1. You can send your PostHog data [to a 3rd party CDP](#sending-posthog-data-to-a-3rd-party-cdp-for-extra-destinations).
-2. Build your own [`composeWebhook` destination](/docs/cdp/build).
-3. Fully transition to a 3rd party CDP (option 2).
+1. Build your own in-app destination [real-time destinations](/docs/cdp/destinations) by writing some custom logic to call the 3rd party API.
+2. You can send your PostHog data [to a 3rd party CDP](#sending-posthog-data-to-a-3rd-party-cdp-for-extra-destinations).
+2. Fully transition to a 3rd party CDP.
```

Comment:
The only way to do this will be to use real time exports so might as well just be one option
```suggestion
2. Fully transition to a 3rd party CDP.
```